content/en/docs/configuring-sdk/_index.md

```diff
@@ -67,19 +67,250 @@ export AWS_REGION=us-west-2
+associating each set of access keys with a profile. For example, in your
+credentials file, you can declare multiple profiles, as follows.
+
+```ini
+[default]
+aws_access_key_id = <YOUR_DEFAULT_ACCESS_KEY_ID>
+aws_secret_access_key = <YOUR_DEFAULT_SECRET_ACCESS_KEY>
+
+[test-account]
+aws_access_key_id = <YOUR_TEST_ACCESS_KEY_ID>
+aws_secret_access_key = <YOUR_TEST_SECRET_ACCESS_KEY>
+
+[prod-account]
+; work profile
+aws_access_key_id = <YOUR_PROD_ACCESS_KEY_ID>
+aws_secret_access_key = <YOUR_PROD_SECRET_ACCESS_KEY>
+```
+
+By default, the SDK checks the `AWS_PROFILE` environment variable to
+determine which profile to use. If no `AWS_PROFILE` variable is set,
+the SDK uses the default profile.
+
+If you have an application named `myapp` that uses the SDK, you can
+run it with the test credentials by setting the variable to
+`test-account myapp`, as shown in the following command.
+
+```bash
+$ AWS_PROFILE=test-account myapp
+```
+
+You can also use instruct the SDK to select a profile by either
+`os.Setenv("AWS_PROFILE", "test-account")` before calling `config.LoadDefaultConfig`,
+or by passing an explicit profile as an argument as shown in the following example:
+
+```go
+cfg, err := config.LoadDefaultConfig(config.WithSharedConfigProfile("test-account"))
+```
+
+{{% pageinfo color="info" %}}
+If you specify credentials in environment variables, the SDK
+will always use those credentials, no matter which profile you specify.
+{{% /pageinfo %}}
+
+### Environment Variables
+
+By default, the SDK detects AWS credentials set in your environment and
+uses them to sign requests to AWS. That way you don't need to manage
+credentials in your applications.
+
+The SDK looks for credentials in the following environment variables:
+
+* `AWS_ACCESS_KEY_ID`
+* `AWS_SECRET_ACCESS_KEY`
+* `AWS_SESSION_TOKEN` (optional)
+
+The following examples show how you configure the environment variables.
+
+#### Linux, OS X, or Unix
+```bash
+$ export AWS_ACCESS_KEY_ID=YOUR_AKID
+$ export AWS_SECRET_ACCESS_KEY=YOUR_SECRET_KEY
+$ export AWS_SESSION_TOKEN=TOKEN
+```
+
+#### Windows
+
+```batch
+> set AWS_ACCESS_KEY_ID=YOUR_AKID
+> set AWS_SECRET_ACCESS_KEY=YOUR_SECRET_KEY
+> set AWS_SESSION_TOKEN=TOKEN
+```
+
+### Specify Credentials Programmatically
+`config.LoadDefaultConfig` allows you to provide an explicit
+[aws.CredentialProvider]({{< apiref "aws#CredentialsProvider" >}}) when loading the shared configuration sources.
+To pass an explicity credential provider when loading shared configuration use
+[config.WithCredentialsProvider]({{< apiref "config#WithCredentialsProvider" >}}). For example, if `customProvider`
+references an instance of `aws.CredentialProvider` implementation, it can be passed during configuration loading
+like so:
+
+```go
+cfg, err := config.LoadDefaultConfig(config.WithCredentialsProvider(customProvider))
+```
+
+A credential provider passed in this manner will be exclusivly be used by the SDK, and no other credentials will be 
```

Comment:
This is in passive voice. How about:

If you explicitly provide credentials, as in this example, the SDK uses only those credentials.